### DISCUSSION TITLE
Welcome to the Cucumber discussion board

### DISCUSSION THREAD

**Discussion by mpkorstanje** [cucumber/common#2170] [General]
Hey folks, welcome to the discussion board for Cucumber!

Feel free to post! But do keep [the Code of Conduct](https://github.com/cucumber/common/blob/main/CODE_OF_CONDUCT.md) in mind.

Cheers!

**Comment by george-gayton** [comment]
Hello!

I'm interested in contributing! Namely I'd like to add Clojure support to the Cucumber language server.

Is this feasible and is there support available for doing so?

Cheers!

**Reply by mpkorstanje** [reply]
You're certainly welcome to try. Unfortunately no-one in the current core team has any expertise in the VS Code ecosystem. So we can't provide support for that at all. For other topics, you're welcome to ask in the Discord or Github Discussions.

We did archive [`cucumber-jvm-clojure`](https://github.com/cucumber-attic/cucumber-jvm-clojure) about 8 years because no-one was contributing to it. I don't remember the exact download statistics, but they weren't great either.  

So I'm curious, what would Clojure support actually target?

**Reply by george-gayton** [reply]
Lambdaisland have an implementation [here](https://github.com/lambdaisland/kaocha-cucumber). I haven't investigated into contributing to any vs code extensions within that project.

**Reply by george-gayton** [reply]
AFAIK they do not have any similar language server implementations to contribute to.

**Reply by mpkorstanje** [reply]
Ah interesting.  Feel free to give it a try.  I had a quick look at the code of the language-service and it seems like you can plugin more languages without them interfering with each other.

**Reply by george-gayton** [reply]
I've submitted this [PR](https://github.com/cucumber/language-service/pull/271) - is there an estimated timeline for review?

**Reply by mpkorstanje** [reply]
No. But I did start the pipe line and saw that the tests weren't passing and paid no further attention on the assumption that you still had work to do. I see that is intentional now.

**Reply by p14n** [reply]
I've submitted a new [PR](https://github.com/cucumber/language-service/pull/309/) that I think addresses the issue with this previous one - would someone be able to take a look?

**Comment by rinollod** [comment]
Hi guys, 
I see that cucumber it totally free, and this is awesome 🥇  
But, I'm not sure if is free only for a certain number of users/license. Is there a limitation in user quantity? Is there any plugin to pay to have more feature?
please guys let me know asap if possible 👍

**Comment by spearhead-ag** [comment]
Hi team,
Researching here for possible use of Cucumber in healthcare settings, is it compatible with Gherkin and does it support CDA-based and FHIR-based clinical document validation, and what tools works best for reporting?
Thanks

or if anyone is in that current space/setting, would like to connect
